Versioning_Engine/Modify/AddVersion.cs

```diff
@@ -37,20 +37,20 @@ public static partial class Modify
         public static void AddVersion(this BsonDocument document)
         {
             // TODO: Uncomment this code after producing the 4.1 beta
```

Comment:
Not to be picky, but could we remove the comment if this PR is resolving the `TODO`? :smile:

Reply:
Good point 🤦